projects/scion/toolkit/uuid/src/uuid.util.ts

```diff
@@ -9,26 +9,36 @@
  */
 
 /**
- * Allows generating pseudo-random identifiers.
+ * Fast UUID generator, RFC4122 version 4 compliant.
+ *
+ * Usage: UUID.randomUUID()
  */
-export class UUID {
+function* generateUUID(): IterableIterator<string> {
+  const lut = [];
+  for (let i = 0; i < 256; i++) {
+    lut[i] = (i < 16 ? '0' : '') + (i).toString(16);
+  }
+
+  let uuid;
```

Comment:
Why do you need to remember the UUID? Can't you just assign it to a `const` in the `while` loop?
```ts
const uuid = lut[d0 & 0xff] + ...
```

Reply:
Good idea. Done.